monkey/monkey_island/cc/ui/src/components/attack/techniques/T1107.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import '../../../styles/Collapse.scss'
+import ReactTable from "react-table";
+import { renderMachineFromSystemData } from "./Helpers"
+
+
+class T1107 extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  static renderDelete(status){
+    if(status === 2){
```

Comment:
We should have consts for technique statuses on a JS file somewhere and use them instead of 0,1,2

Reply:
Refactored the whole thing to be more consistent: we return status as int both to the mongodb and to front end.